### ISSUE TITLE
PingURL for deployable

### ISSUE THREAD

**Issue by voitau** [bmuschko/gradle-cargo-plugin#75]
Cargo maven2 plugin has this option:

 pingURL

"URL on which to ping the deployed or undeployed application (to check if deployment or undeployment is successful), that should return an HTTP OK response only after the deployment is complete. If not set, the deployed or undeployed application will not be pinged, hence the deployment considered as complete as soon as the target server's method returns successfully."

Source: http://cargo.codehaus.org/Maven2+Plugin+Reference+Guide

Nice to have feature. For JBoss 7 cargoStartLocal finishes as soon as app server is started and is not waiting for deployment to complete. pingURL helps to wait for deployment.

**Comment by sohag245** [comment]
https://newsworldall011.blogspot.com/2023/12/how-each-us-house-member-voted-on.html
